server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/formula/CounterInitializationContext.java

```diff
@@ -42,7 +42,19 @@
 
   /**
    * Lists of Periods defined for the current project. They can be used to retrieve variations Measure.
+   * @deprecated replaced by {@link #getPeriod()}
    */
+  @Deprecated
   List<Period> getPeriods();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)